CsCore/xUnitTests/src/Plugins/CsCoreXUnitTests/com/csutil/tests/http/apis/OpenAiTests.cs

```diff
@@ -199,6 +201,76 @@ public enum Emotion { happy, sad, angry }
 
         }
 
+
+        [Fact]
+        public async Task ExampleTTSandSTT() {
+            string textToTest = "hello world";
+            Stream TTSResponseStream = await TTS(new Audio.TTSRequest() { input = textToTest });
+            Assert.NotNull(TTSResponseStream);
+
+            Audio.STTResponse STTResponse = await STT(new Audio.STTRequest() { file = TTSResponseStream });
+            Assert.NotEmpty(STTResponse.text);
+            Log.d(STTResponse.text);
+            //fails when textToTest contains numbers
+            Assert.Equal(formatString(STTResponse.text), formatString(textToTest));
+        }
+
+        private string formatString(string str) {
```

Comment:
add docu that this removes all characters from the string except letters, digits and whitespaces

Reply:
Does the test fail if you only do a ToLower of the two strings and not do the other cleanups?

Reply:
1. Comment added. 
2. yes the test fails for example with "hello my name is jack" input and output is "hello, my name is Jack"